### PR TITLE
Dynamic Simulation: translation for certain (tooltip) labels in result UI

### PR DESCRIPTION
**Pull request by thangqp**
**Demo some translations**:

result_translation

**Bonus**: remove auto-switch to full view mode when click on 'switch to full graph' mode

### CODE REVIEW

File: src/components/results/dynamicsimulation/dynamic-simulation-result-chart.js [hunk 1]
```diff
@@ -289,11 +300,6 @@ const DynamicSimulationResultChart = ({
 
             // set the current plot id in scaling
             setPlotIdScale(plotScale ? plotId : undefined);
-
-            // auto switch in full view
-            if (plotScale) {
-                setFullView(plotScale);
-            }
```
Comment (EstherDarkish) [inline src/components/results/dynamicsimulation/dynamic-simulation-result-chart.js:296]: Is this related to the translations ?

Comment (thangqp) [reply src/components/results/dynamicsimulation/dynamic-simulation-result-chart.js:296]: No, this is a tiny bonus which is noted the the bottom of description..
Why i have to do this in the PR? for making be more coherent to the label 'Enlarge the graph'

File: src/translations/fr.json [hunk 2]
```diff
@@ -191,6 +191,21 @@
     "DynamicSimulationResultTab": "Page",
     "DynamicSimulationResultChart": "Graphe",
     "DynamicSimulationResultJson": "Données",
+    "DynamicSimulationSyncPlotEnable": "Cliquez pour désactiver la synchronisation des actions",
+    "DynamicSimulationSyncPlotDisable": "Cliquez pour activer la synchronisation des actions",
+    "DynamicSimulationCloseTab": "Fermer l'onglet",
+    "DynamicSimulationAddTab": "Ajouter un onglet",
+    "DynamicSimulationAddGraph": "Ajouter un graphe",
+    "DynamicSimulationCloseGraph": "Fermer le graphe",
+    "DynamicSimulationPlotScaleEnable": "Agradir le graphe",
```
Comment (EstherDarkish) [inline src/translations/fr.json:200]: Agradir => Agrandir

File: src/translations/fr.json [hunk 3]
```diff
@@ -191,6 +191,21 @@
...
+    "DynamicSimulationPlotScaleDisable": "Restaurer le graphe",
+    "DynamicSimulationFullViewEnable": "Agrandir la vue de résultat",
+    "DynamicSimulationFullViewDisable": "Restaurer la vue de résultat",
+    "DynamicSimulationShowSeriesListDisable": "Cacher la list de courbes",
```
Comment (EstherDarkish) [inline src/translations/fr.json:204]: list => liste

**Review by EstherDarkish — APPROVED** [review]
Test OK, code OK, be sure to fix some french translation issues before merging.

File: src/translations/fr.json (hunk 2 above)
Comment (thangqp) [reply src/translations/fr.json]: Done!

File: src/translations/fr.json (hunk 3 above)
Comment (thangqp) [reply src/translations/fr.json]: Done
